server/schemas/item.graphql

```diff
@@ -10,6 +10,7 @@ type Item implements Node {
   user: User
   schema: Schema!
   model: Model!
+  status: [ItemStatus!]!
```

Comment:
I don't sure that why one item has multiple status

Reply:
as mentioned in the description item status might be `public` and `in review`

Reply:
```suggestion
  status: ItemStatus!
```